EcpBtBot/DialogFlow/Response.py

```diff
@@ -19,7 +19,7 @@ def status(self):
 class Result:
```

Comment:
the name of the file should be in lower case (following the python standard.) to know better about them: https://www.python.org/dev/peps/pep-0008/. It's always good to follow those.